.changeset/shy-files-fix.md

```diff
@@ -0,0 +1,5 @@
+---
+'@sumup/cra-template': patch
+---
+
+Added default options to initialize Foundry.
```

Comment:
Do we need changesets for the changes to the CNA foundry script, or for the workflows?

Reply:
No. The CRA template needs it to  trigger a publish to NPM. I guess we could add a changeset for CNA as well, just for documentation. The workflows are not user facing changes.